tests/test_fluidsynth.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module contains tests for the fluidsynth methods.
+"""
+import unittest
+
+import numpy as np
+from scipy.io import wavfile
+import tempfile
+
+from partitura.utils.fluidsynth import (
+    synthesize_fluidsynth,
+    HAS_FLUIDSYNTH,
+    SAMPLE_RATE,
+)
+
+from partitura import EXAMPLE_MUSICXML, load_score, load_performance_midi
+
+from partitura.io.exportaudio import save_wav_fluidsynth
+
+from tests import MOZART_VARIATION_FILES
+
+RNG = np.random.RandomState(1984)
```

Comment:
1984 seems like a weird RandomState.... is that symbolic? Not a important comment

Reply:
Well... I have used 1984 as my default random state for more than a decade (42 felt very obvious 😉). It is a reference to George Orwell's 1984 (since the book describes a totalitarian state, and I wanted my default rng to be everywhere) 🤓